buildbot/master/files/config/steps.yml

```diff
@@ -76,3 +76,12 @@ arm64:
   - ./mach build --rel --target=aarch64-unknown-linux-gnu
   - bash ./etc/ci/lockfile_changed.sh
   - bash ./etc/ci/manifest_changed.sh
+
+windows:
+  - ./mach build -d -v
```

Comment:
Let's do `./mach build --dev` for consistency. Also, please rename to `windows-dev` as well.